documents/policies/Competition Requirements.md

```diff
@@ -0,0 +1,141 @@
+         1. If any location of such competitions is closer than 100 km to any other competition, then
+            1. both competitions should be at least 5 days apart, or
+            2. the other competition must not have the 3x3x3 Fewest Moves event.
+      2. A 3x3x3 Fewest Moves simultaneous competition cannot be part of any other regular competition.
+         1. Time frames when both are held cannot overlap.
+   5. Twin competitions
+      1. Twin competitions are WCA Competitions that are held less than 5 days apart for which the distance between them is less than 100 km, with the intention to accommodate more competitors to compete in one area.
+         1. Twin competitions may be accepted if the need for competitions in an area is so great that the community will benefit from having multiple competitions held in a short period of time.
+      2. Twin competitions are obliged to clearly and publicly state their condition of twin competitions and their special participation terms.
+         1. A competitor is only permitted to participate in one of the twin competitions.
+            1. If a competitor competes at more than one of the involved twin competitions, only the results from the competition that they compete in first will be kept.
+3. WCA Delegate(s)
+4. Organizer(s)
+   1. Organizers can be specific persons and/or organizations.
+5. Competitor limit
+   1. The competitor limit can be on either a "first come first serve" basis or based upon qualification results or rankings in the WCA world rankings of a previously announced calendar date.
+      1. Qualification for an event must be based only on results from the same event.
+   2. If the competitor limit for a competition or an event is reached before the competition, the organization team must not approve any additional registrations for the competition or the event.
+   3. The competitor limit is per competition. Per day competitor limits are not allowed.
+6. Registration requirements
+   1. The registration requirements must be clear so that any competitor from anywhere in the world knows how to and has the ability to register.
+      1. The registration requirements must be available in English.
+      2. In addition to the English version, the registration requirements can be available in any other language and should be available in the local language.
+   2. The registration period(s) must be a clearly defined time range with a start and end time and date.
+      1. The first registration period must have a start time and date at least 48 hours after the competition is announced.
+         1. The competition's Delegate(s) and/or organizers may register for the competition prior to the first registration period.
+      2. If there is more than one registration period, the organization team must announce each additional registration period at least 48 hours before it starts.
```

Comment:
1. and 2. can be combined:
```suggestion
      1. Each registration period must be announced publicly on WCA at least 48 hours before it starts.
```
But I'm also fine keeping it the way it is now for clarity.

Reply:
I think that's a good suggestion, I will add it when pushing all the changes!